### DISCUSSION TITLE
PM-Tiles vs PBF-Tiles Comparison

### DISCUSSION THREAD

**Discussion by koufopoulosf** [protomaps/PMTiles#189] [Q&A] [answered]
Hello all,

I am making this issue, as I would like your valuable opinion on the following matter.

So far, for making a basemap (not relying on a 3rd party API), we have to worry about provisioning a server, scaling it, maintaning it, and all the fun that comes with it (like in the case of MBTiles). However, there are two alternative ways, that I would consider.

One is to use protomaps, the so called PMTiles, which essentially is a simple blob that the frontend will download (only desired parts) and render directly in the browser.

The other suggestion would be to use tippecanoe with the ```--output-to-directory``` flag to output the individual .pbf tiles instead of a single MBTiles file. These tiles can then be hosted on something like S3 and your map-rendering client can query the necessary tiles, no server required.

So my question would be:

Has anyone experience with both of these methods for deploying a scalable basemap and/or is there any other suggestion that could be considered efficient in terms of serving a basemap without the need of a server?

Thanks in advance for your valuable time.

Best Regards,

Filippos

**Comment by bdon** [comment, marked as answer]
The question about using PMTiles as a container format instead of hosting raw tiles in a directory is addressed here: 

~https://protomaps.com/docs/faq#why-not-store-individual-tiles-on-s3~

https://docs.protomaps.com/pmtiles/#individual-files

**Reply by koufopoulosf** [reply]
> The question about using PMTiles as a container format instead of hosting raw tiles in a directory is addressed here:
> 
> https://protomaps.com/docs/faq#why-not-store-individual-tiles-on-s3

Great, thank you very much!

**Reply by do-me** [reply]
fyi: link is broken

**Reply by bdon** [reply]
updated the link above
